autoload/widgets/fzf-named-directories

```diff
@@ -0,0 +1,8 @@
+#!/bin/zsh
+
+hash -d | sed "s/\([^=]*\)/\x1b[34;1m\1\x1b[0m/" | \
+  __fzf::widget::select $FZF_WIDGET_OPTS[change-dir] +m --ansi | \
```

Comment:
Change `$FZF_WIDGET_OPTS[change-dir]` to `$FZF_WIDGET_OPTS[change-named-directory]`.